cli/internal/taskhash/taskhash.go

```diff
@@ -344,3 +349,14 @@ func (th *Tracker) GetExpandedInputs(packageTask *nodes.PackageTask) map[turbopa
 
 	return inputsCopy
 }
+
+// GetEnvVars returns the hashed env vars for a given taskID.
+// This operation is protected by a mutex, because the underlying map where the
```

Comment:
As above, I think this comment is better attached to the declaration of the map. The fact that it is protected by the mutex is a property of the map, not of this particular method.